weblate/trans/views/dictionary.py

```diff
@@ -334,9 +351,11 @@ def show_dictionary(request, project, lang):
             'language': lang,
             'page_obj': words,
             'form': form,
-            'query_string': 'letter={}'.format(letter) if letter else '',
+            'query_string': query_string,
             'uploadform': uploadform,
             'letterform': letterform,
+            'searchform': searchform,
+            'search': search,
```

Comment:
Is this variable used somewhere in the template?

Reply:
Yes, it is the value of the `term` input field.

Reply:
Ah, but that should rendered though form, not manually.